packages/hooks/src/useLockScroll.ts

```diff
@@ -42,8 +42,12 @@ const setStyleProperty = (
   }
 }
 
-export const useLockScroll = (element: HTMLElement = document.body): void => {
+const useIsomorphicLayoutEffect =
```

Comment:
Looks like a useful stuff, can we put it in a separate file and export from package?

Reply:
Sure, will do.

As for the PR itself, I. guess it's okay to close it. Doesn't seem like a good solution, as we've discussed.